Source/LanguageModelManager.mm

```diff
@@ -305,4 +305,28 @@ + (void)setPhraseReplacementEnabled:(BOOL)phraseReplacementEnabled
     gLanguageModelMcBopomofo.setPhraseReplacementEnabled(phraseReplacementEnabled);
 }
 
++ (nullable NSString *)pronounciationFor:(NSString *)phrase {
+    if (!gLanguageModelMcBopomofo.isDataModelLoaded()) {
+        [self loadDataModel:InputModeBopomofo];
+    }
+
+    std::vector<std::string_view> records = gLanguageModelMcBopomofo.getPronounciations(phrase.UTF8String);
```

Comment:
I already made a comment on the issue with using vector of string views, so I'm going to focus on the design issue here.

Namely, here the internal knowledge of the LM ("a record raw has three columns") is leaked into the call site here. Wouldn't it be better if the parsing is done in ParselessLM? This will mean you'll want to define a data structure for reverse lookup though (how about a struct like `FoundReading` with two fields, `reading` and `score`?), but then this method can just go through the returned vector of the found readings without having to have the (leaky and potentially fragile) knowledge of splitting the record rows on its own.